index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+    <head> 
+        <title>Kalina Gardziała - CV</title>
+        <link rel="stylesheet" href="style.css" /> 
+    </head>
+     <body>
+        <div class="container">
+            <header> 
+                <img src="ja.jpg" alt= "My Profile Picture"/> 
+                <h1>Kalina Gardziała</h1>
+                <address>
+                    <ul> 
+                        <li> <img class="address-icon circle" src="icons/phone.png"/> <b>Telephon</b> 787 779 021</li>
+                        <li> <img class="address-icon circle" src="icons/mail.png"/> <b>Email</b> kalinagardziala@gmail.com</li>
+                    </ul>
```

Comment:
You don't need here circle class. You can use border-radius property in address-icon class